src/github/client/github-queries.ts

```diff
@@ -218,6 +218,20 @@ export type getBranchesResponse = {
 	}
 };
 
+export type getBranchesNameResponse = {
+	repository: {
+		refs: {
+      totalCount: number,
```

Comment:
Can drop total count now

Reply:
It may be helpful if we want to tweak UI (display message or make drop down editable) if branches are more than 100